### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by munyite001**
Add new images for modal in assets folder

Add structure for modal in html just as a placeholder

Add styling for modal

Add functionality to dynamically fill the modal content for each project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -94,7 +143,7 @@ <h3 class="sub-heading">Multi-Post Stories</h3>
               <li class="lang-tag">bootstrap</li>
```
Comment (HAMMAS-SALEEM) [inline index.html:143]: - [ ] Kindly remove your project cards from your html and generate them dynamically from your javascript file to follow the requirements :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @munyite001 :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Descriptive PR Title
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Following Linters
:heavy_check_mark: Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (munyite001) [reply index.html:143]: did that and I explained that I commented it out

**Review by KDlamini — APPROVED** [review]
### Hi @munyite001 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Design follows FIGMA guidelines
✔️ Modal works as expected
✔️ Good PR title and description
✔️ No linter errors

# Status: Approved ✔️ 

simon

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**